tests/tests_selinux_modules.yml

```diff
@@ -90,7 +88,7 @@
           include_role:
             name: linux-system-roles.selinux
       when: ansible_distribution_major_version < "7" and
-        ( ansible_distribution == "CentOS" or ansible_distribution == "RedHat" )
+        ansible_distribution in ["CentOS", "RedHat"]
```

Comment:
This isn't applicable for Rocky?

Reply:
I did not check the details why the test was originally only implemented for `ansible_distribution_major_version < "7"`.
Rocky was not added here because its initial release was Rocky Linux 8.4 (https://rockylinux.org/news/rocky-linux-8-4-ga-release/) and therefore `ansible_distribution_major_version == "8"`.